src/c/types/sequence.c

```diff
@@ -182,6 +200,12 @@ bool serialize_endian_sequence_uint64_t(MicroBuffer* buffer, const Endianness en
     return serialize_endian_array_uint64_t(buffer, endianness, array, size);
 }
 
+bool serialize_endian_sequence_int8_t(MicroBuffer* buffer, const Endianness endianness, const int8_t* array, const uint32_t size)
+{
+    serialize_endian_uint32_t(buffer, endianness, size);
```

Comment:
This function is not safe. First call can fail but second not. Example: Array of one element when buffer is only 3 bytes left.

Reply:
Done!